pkg/controllers/builder.go

```diff
@@ -384,26 +393,26 @@ func getStorageClassName(vdb *vapi.VerticaDB) *string {
 }
 
 // buildStsSpec builds manifest for a subclusters statefulset
-func buildStsSpec(nm types.NamespacedName, vdb *vapi.VerticaDB, scHandle *SubclusterHandle) *appsv1.StatefulSet {
+func buildStsSpec(nm types.NamespacedName, vdb *vapi.VerticaDB, sc *vapi.Subcluster) *appsv1.StatefulSet {
 	return &appsv1.StatefulSet{
 		ObjectMeta: metav1.ObjectMeta{
 			Name:        nm.Name,
 			Namespace:   nm.Namespace,
-			Labels:      makeLabelsForObject(vdb, scHandle),
+			Labels:      makeLabelsForObject(vdb, sc),
 			Annotations: makeAnnotationsForObject(vdb),
 		},
 		Spec: appsv1.StatefulSetSpec{
 			Selector: &metav1.LabelSelector{
-				MatchLabels: makeSvcSelectorLabels(vdb, scHandle),
+				MatchLabels: makeSvcSelectorLabels(vdb, sc),
```

Comment:
Is sticking to sc instead of scHandle only due to the fact that vapi.Subcluster is already passed around?

Reply:
Yes, so this undoes some of the work that I did earlier.  Overall, I think it makes it simpler.

Reply:
Ok got it.